specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -256,6 +256,45 @@
           "type": "object"
         }
       }
+    },
+    "StorageLifecyclePolicyCompletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Storage.LifecyclePolicyCompleted event.",
+      "type": "object",
+      "properties": {
+        "scheduleTime": {
+          "description": "The time the policy task was scheduled.",
+          "type": "string"
+        },
+        "deleteSummary": {
+          "$ref": "#/definitions/StorageLifecyclePolicyActionSummaryDetail"
+        },
+        "tierToCoolSummary": {
+          "$ref": "#/definitions/StorageLifecyclePolicyActionSummaryDetail"
+        },
+        "tierToArchiveSummary": {
+          "$ref": "#/definitions/StorageLifecyclePolicyActionSummaryDetail"
+        }
+      }
+    },
+    "StorageLifecyclePolicyActionSummaryDetail": {
+      "description": "Action execution stats.",
```

Comment:
Typo?

Do you mean status?

Also, please consider add more details for the description here

Reply:
Modified